### ISSUE TITLE
Fatal error on first use of plugin: `Spell Checker`

### ISSUE THREAD

**Issue by jimrandell** [Flow-Launcher/Flow.Launcher#1285]
Hi,
Thank you for your work on this launcher...with plugins and sort priority tweaks, it's getting closer to Spotlight all the time!

### **Fatal error using plugin: `Spell Checker`!**

****I just installed plugin: `Spell Checker`, opened FlowLauncher,**** 
- typed in a word i would commonly misspell
  - don't remember now what it was
    - went back to try again, 
    - to test a another word, so I could include it here
- opened FL, typed `spell `...and FL immediately closed
- changed Action Word to `sp`, 
  - same results
  - typed `sp ` and FL died

**Following instructions in error message, I have...:**
- created this issue.
- included this file:
  - [C:\Users\jimr\AppData\Roaming\FlowLauncher\Logs\1.9.3\2022-07-20.txt](https://github.com/Flow-Launcher/Flow.Launcher/files/9148228/2022-07-20.txt) 
- pasted the following error:
```Python 
Flow Launcher version: 1.9.3
OS Version: Microsoft Windows NT 10.0.19044.0
IntPtr Length: 8
x64: True

Python Path: 
Date: 07/20/2022 09:36:51
Exception:
System.IO.InvalidDataException: Spell Checker|Traceback (most recent call last):
  File "C:\Users\jimr\AppData\Roaming\FlowLauncher\Plugins\Spell Checker-0.0.5\run.py", line 9, in <module>
    from plugin.main import SpellChecker
  File "C:\Users\jimr\AppData\Roaming\FlowLauncher\Plugins\Spell Checker-0.0.5\plugin\main.py", line 6, in <module>
    from textblob import Word, TextBlob
  File "C:\Users\jimr\AppData\Roaming\FlowLauncher\Plugins\Spell Checker-0.0.5\lib\textblob\__init__.py", line 2, in <module>
    from .blob import TextBlob, Word, Sentence, Blobber, WordList
  File "C:\Users\jimr\AppData\Roaming\FlowLauncher\Plugins\Spell Checker-0.0.5\lib\textblob\blob.py", line 29, in <module>
    import nltk
  File "C:\Users\jimr\AppData\Roaming\FlowLauncher\Plugins\Spell Checker-0.0.5\lib\nltk\__init__.py", line 137, in <module>
    from nltk.text import *
  File "C:\Users\jimr\AppData\Roaming\FlowLauncher\Plugins\Spell Checker-0.0.5\lib\nltk\text.py", line 29, in <module>
    from nltk.tokenize import sent_tokenize
  File "C:\Users\jimr\AppData\Roaming\FlowLauncher\Plugins\Spell Checker-0.0.5\lib\nltk\tokenize\__init__.py", line 65, in <module>
    from nltk.tokenize.casual import TweetTokenizer, casual_tokenize
  File "C:\Users\jimr\AppData\Roaming\FlowLauncher\Plugins\Spell Checker-0.0.5\lib\nltk\tokenize\casual.py", line 49, in <module>
    import regex  # https://github.com/nltk/nltk/issues/2409
  File "C:\Users\jimr\AppData\Roaming\FlowLauncher\Plugins\Spell Checker-0.0.5\lib\regex\__init__.py", line 1, in <module>
    from .regex import *
  File "C:\Users\jimr\AppData\Roaming\FlowLauncher\Plugins\Spell Checker-0.0.5\lib\regex\regex.py", line 419, in <module>
    import regex._regex_core as _regex_core
  File "C:\Users\jimr\AppData\Roaming\FlowLauncher\Plugins\Spell Checker-0.0.5\lib\regex\_regex_core.py", line 20, in <module>
    import regex._regex as _regex
ModuleNotFoundError: No module named 'regex._regex'

   at Flow.Launcher.Core.Plugin.JsonRPCPlugin.ExecuteAsync(ProcessStartInfo startInfo, CancellationToken token) in C:\projects\flow-launcher\Flow.Launcher.Core\Plugin\JsonRPCPlugin.cs:line 287
   at Flow.Launcher.Core.Plugin.JsonRPCPlugin.ExecuteAsync(ProcessStartInfo startInfo, CancellationToken token) in C:\projects\flow-launcher\Flow.Launcher.Core\Plugin\JsonRPCPlugin.cs:line 301
   at Flow.Launcher.Core.Plugin.JsonRPCPlugin.QueryAsync(Query query, CancellationToken token) in C:\projects\flow-launcher\Flow.Launcher.Core\Plugin\JsonRPCPlugin.cs:line 321
   at Flow.Launcher.Core.Plugin.PluginManager.<>c__DisplayClass22_0.<<QueryForPluginAsync>b__0>d.MoveNext() in C:\projects\flow-launcher\Flow.Launcher.Core\Plugin\PluginManager.cs:line 192
--- End of stack trace from previous location ---
   at Flow.Launcher.Infrastructure.Stopwatch.DebugAsync(String message, Func`1 action) in C:\projects\flow-launcher\Flow.Launcher.Infrastructure\Stopwatch.cs:line 40
   at Flow.Launcher.Core.Plugin.PluginManager.QueryForPluginAsync(PluginPair pair, Query query, CancellationToken token) in C:\projects\flow-launcher\Flow.Launcher.Core\Plugin\PluginManager.cs:line 211
   at Flow.Launcher.ViewModel.MainViewModel.<>c__DisplayClass137_0.<<QueryResults>g__QueryTask|2>d.MoveNext() in C:\projects\flow-launcher\Flow.Launcher\ViewModel\MainViewModel.cs:line 623
--- End of stack trace from previous location ---
   at Flow.Launcher.ViewModel.MainViewModel.QueryResults() in C:\projects\flow-launcher\Flow.Launcher\ViewModel\MainViewModel.cs:line 624
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__140_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
'''

**Comment by deefrawley** [comment]
That keyword and error is from the Spell Checker plugin, not Dictionary. Did you let Flow install Python for you or are you using a pre-existing install on your PC?

**Comment by jimrandell** [comment]
Well color me embarrassed! (...or dyslexic, can't really tell as I'm also color blind).

 - I'm using a pre-existing install of Python 3.10.3
   - installed on the root of C:\\
   - is there more info I can provide on it? 
 - I didn't mean to be contrarian about it, 
   - just trying to reduce the number of such installs. 

I would be fine with using the Flow install of python, if that just makes it significantly easier to support... just let me know.

I'd like you know how much I appreciate the time you find to put into these efforts!

Cheers,
-Me

**Comment by deefrawley** [comment]
It could be something Python 3.10 has changed. Garulf is the plugin dev so I'm sure he'll have a solution soon.

**Comment by Garulf** [comment]
Currently my plugins are targeted at Python 3.8 (this is the version Flow Launcher uses by default).

3.10 added a few breaking changes (very odd changes) that I have fixed but haven't updated every plugin for just yet.

**Comment by jjw24** [comment]
@Garulf has this plugin received the update to address this error?

**Comment by Garulf** [comment]
The latest update includes fixes for 3.10 so yes all issues should be addressed.

**Comment by jjw24** [comment]
Ok thank you.
